signal/signal.go

```diff
@@ -26,7 +27,11 @@ var (
 )
 
 func init() {
-	signal.Notify(interruptChannel, os.Interrupt)
+	signalsToCatch := []os.Signal{
+		os.Interrupt, os.Kill, syscall.SIGABRT, syscall.SIGTERM,
+		syscall.SIGSTOP, syscall.SIGQUIT,
```

Comment:
style nit: new line for each signal?

Reply:
Fixed, also we'll now log the signal itself.